src/libostree/ostree-sysroot.c

```diff
@@ -1108,6 +1108,38 @@ find_booted_deployment (OstreeSysroot       *self,
 }
 
 /**
+ * ostree_sysroot_get_last_deployed:
+ * @self: Sysroot
+ * @osname: (allow-none): Operating system group
+ *
+ * Find the last deployed sysroot.
+ *
+ * Returns: (transfer full): The last deployment in OS or NULL if none
+ */
+OstreeDeployment *
+ostree_sysroot_get_last_deployed (OstreeSysroot     *self,
```

Comment:
I'd call this something like `ostree_sysroot_get_default_deployment`?

Actually, we have a similar version of this in rpm-ostree that returns not just the pending deployment, but also the rollback:

https://github.com/projectatomic/rpm-ostree/blob/95e9aa4/src/daemon/rpmostree-sysroot-core.c#L399

Maybe we should just upstream that to libostree.